digiroad2-oracle/src/main/resources/db/migration/V213__add_table_dataset.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE MUNICIPALITY_DATASET (
+ DATASET_ID CHAR(36) PRIMARY KEY,
+ GEOJSON CLOB NOT NULL,
+ ROADLINKS CLOB NOT NULL,
+ RECEIVED_DATE TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL,
+ PROCESSED_DATE TIMESTAMP,
+ STATUS NUMBER
+);
+
+CREATE TABLE MUNICIPALITY_FEATURE (
+ FEATURE_ID NUMBER,
+ DATASET_ID CHAR(36) REFERENCES MUNICIPALITY_DATASET(DATASET_ID),
+ STATUS VARCHAR2(4000),
+ CONSTRAINT PK_MUNICIPALITY_FEATURE PRIMARY KEY(FEATURE_ID, DATASET_ID)
+);
```

Comment:
In my opinion we don't need a second table to give the status feature error, the municipality_dataset could support all this logs. For that only needs to include an blob/clob column to write a json with a similar structur
[{ FeatureId: 12 FailedReason: [{"InvalidRoadlink": 523456, "InvalidValue": "value x is not in range of allowed values" }]
[{ FeatureId: 50 FailedReason: [{"MissingData": "Field x is mandatory", "InvalidValue": "value x is not in range of allowed values" }]

Reply:
For me both cases are valid.